blog/2024/10-26-media-segments-10.10.0/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Jellyfin Media Segments Explained 10.10.0
+description: What are Media Segments and how to use them in version 10.10.0
+authors: JPVenson
+slug: jellyfin-media-segments-10.10.0
+tags: [release]
+---
+
+First of all things i would like to introduce myself as a new Server-Team member for the jellyfin FOSS project.
```

Comment:
While I appreciate the introduction, I'd leave it for the end :-) Also capitalization, etc.